aspnet/security/authorization/policies.rst

```diff
@@ -63,7 +64,7 @@ A requirement doesn't need to have data or properties.
 Authorization Handlers
 ----------------------
 
-An authorization handler is responsible for the evaluation of any properties of a requirement. The  authorization handler must evaluate them against a provided ``AuthorizationContext`` to decide if authorization is allowed. A requirement can have :ref:`multiple handlers <security-authorization-policies-based-multiple-handlers>`. Handlers must inherit ``AuthorizationHandler<T>`` where T is the requirement it handles. 
+An authorization handler is responsible for the evaluation of any properties of a requirement. The  authorization handler must evaluate them against a provided :dn:class:`~Microsoft.AspNetCore.Authorization.AuthorizationHandlerContext` to decide if authorization is allowed. A requirement can have :ref:`multiple handlers <security-authorization-policies-based-multiple-handlers>`. Handlers must inherit :dn:class:`~Microsoft.AspNetCore.Authorization.AuthorizationHandler<TRequirement>` where ``TRequirement`` is the requirement it handles. 
```

Comment:
Actually this statement is blatantly wrong in our documentation:

"Handlers must inherit `Microsoft.AspNetCore.Authorization.AuthorizationHandler<TRequirement>` where TRequirement is the requirement it handles."

Handlers MAY inherit this base class as a convenience, and the end of the day, but we don't care how exactly the requirements are met, only each requirement must be `.Succeed()` on the context for the authZ to succeed as a whole.